lib/c/libintvector.h

```diff
@@ -2,9 +2,16 @@
 #define __Vec_Intrin_H
 
 #include <sys/types.h>
-#include <wmmintrin.h>
+
+#define Lib_IntVector_Intrinsics_bit_mask64(x) -((x) & 1)
+
```

Comment:
at this stage we should contemplate having a `./configure` script that checks for things such as:
- presence of OCaml and whether to build bindings
- recent enough toolchain that has the avx2 headers, and bail otherwise